packages/kbn-interpreter/common/lib/paths_registry.ts

```diff
@@ -17,52 +17,50 @@
  * under the License.
  */
 
-import clone from 'lodash.clone';
-
 class PathsRegistry {
+  private paths: Map<string, string[]>;
```

Comment:
I've read a bit where this code is executed, this should be a different type signature like
`Map<string, PathLike[]>` using `PathLike` from `fs` lib. the `PathLike` type should be then used in the rest of this file for a better type checking